app/workers/facilities/pssg_download.rb

```diff
@@ -31,28 +32,37 @@ def create_and_save_drive_time_data(drive_time_data)
       name = attributes&.dig('Name')
       drive_time_band = facility.drivetime_bands.find_or_initialize_by(vha_facility_id: id, name: name)
       drive_time_band.unit = 'minutes'
-      drive_time_band.min = attributes&.dig('FromBreak')
-      drive_time_band.max = attributes&.dig('ToBreak')
+      drive_time_band.min = round_band(attributes&.dig('FromBreak'))
+      drive_time_band.max = round_band(attributes&.dig('ToBreak'))
       drive_time_band.name = name
-      drive_time_band.polygon = extract_polygon(drive_time_data)
+
+      drive_time_band.polygon = extract_polygon(rings)
       drive_time_band.save
       facility.save
     end
 
-    def extract_polygon(drive_time_data)
-      rings = drive_time_data&.dig('geometry', 'rings')
+    def round_band(band)
+      if (band % 10).zero?
+        band
+      else
+        (band.to_i / 10) * 10
```

Comment:
```suggestion
        (band.to_i / 10 + 1) * 10
```
I think we need to round up to the next 10 minute increment. Otherwise we are misrepresenting the amount of time it might take someone. For instance rounding 15 minutes down to 10 means that we are saying that anyone in the drivetime band should be able to get to the facility within 10 minutes, which isn't true.